### PR TITLE
gold carryover scenarios1

### PR DESCRIPTION
**Pull request by white-haired-uncle**
20230725 10:17:23 warning deprecation: NEW_GOLD_CARRYOVER_NOTE_40 has been deprecated and may be removed in version 1.17.0.; This is now automatically generated and not required. Use [objectives][gold_carryover]carryover_percentage=40 instead.
20230725 10:17:23 warning deprecation: EARLY_FINISH_BONUS_NOTE has been deprecated and may be removed in version 1.17.0.; This is now automatically generated and not required. Use [objectives][gold_carryover]bonus=yes instead.

Breaking up by chapter for easy review.

I'd like to see these in every scenario, instead of assuming player knows the defaults (which could change).

### CODE REVIEW

**Comment by white-haired-uncle** [comment]
Two hundred scenarios, and one comment...

```
#No idea what the hell is this - but it may have some purpose. Was I drunk when
I was writing this?
```

**Comment by white-haired-uncle** [comment]
Okay, that was a little messy, but part1 is now free of gold carryover messages.

**Comment by Dugy** [comment]
Can I merge it now?

**Comment by white-haired-uncle** [comment]
sure

File: scenarios2/05_Umbra.cfg [hunk 1]
```diff
@@ -217,9 +217,9 @@
                         [set_specials]
                             mode=append
                             {WEAPON_SPECIAL_POISON}
-#textdomain wesnoth-units
+                            #textdomain wesnoth-units
```
Comment (Dugy) [inline scenarios2/05_Umbra.cfg:220]: Unwanted change.

File: scenarios2/05_Umbra.cfg [hunk 2]
```diff
@@ -217,9 +217,9 @@
...
                             {WEAPON_SPECIAL_PLAGUE_TYPE_LOTI Shadow _"Shadow"}
-#textdomain wesnoth-loti
+                            #textdomain wesnoth-loti
```
Comment (Dugy) [inline scenarios2/05_Umbra.cfg:222]: Unwanted change.

**Review by Dugy — CHANGES_REQUESTED** [review]

**Comment by white-haired-uncle** [comment]
Just confirmed it's wmlindent doing that.

File: scenarios2/05_Umbra.cfg (hunk 2 above)
Comment (white-haired-uncle) [reply scenarios2/05_Umbra.cfg]: done

File: scenarios2/05_Umbra.cfg (hunk 1 above)
Comment (white-haired-uncle) [reply scenarios2/05_Umbra.cfg]: done

**Comment by white-haired-uncle** [comment]
And that's the last of them (not incl spinoffs).

I did notice there are some scenarios in ch10 (at least?) where the gold carry over is only set (to 100) at victory.  I believe this would be misleading if the player assumes that no info in objectives means the wesnoth default.

**Comment by Dugy** [comment]
> I believe this would be misleading if the player assumes that no info in objectives means the wesnoth default.

Yes, it could be misleading, it does make sense to add carryover notes about that.

**Comment by white-haired-uncle** [comment]
yes

**Pull request merged**